File: src/B_2751/B_2751.java
package B_2751;
import java.io.*;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collection;
import java.util.Collections;

public class B_2751 {

    public static void main(String[] args) throws IOException {
        StringBuilder sb = new StringBuilder();
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        ArrayList<Integer> arr = new ArrayList<>();
        int a= Integer.parseInt(br.readLine());
        for(int i=0; i<a; i++){
            arr.add(Integer.parseInt(br.readLine()));
        }
        Collections.sort(arr);
        for(int i=0; i<a; i++){
            sb.append(arr.get(i)).append("\n");
        }
        System.out.println(sb);
    }

}

File: src/B_2751/B_2751.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val N = br.readLine().toInt()
    val range = 2000001
    val sb = StringBuilder()
    val num= BooleanArray(range)

    for(i in 0 until N) {
        num[br.readLine().toInt() + 1000000] = true
    }

    for(i in num.indices) {
        if(num[i]) sb.append(i - 1000000).append("\n")
    }
    println(sb)
}